src/rocketchat.coffee

```diff
@@ -153,12 +153,13 @@ class RocketChatBotAdapter extends Adapter
 
 					if curts > @lastts
 						@lastts = curts
+						
+						user = @robot.brain.userForId newmsg.u._id, name: newmsg.u.username
```

Comment:
Out of curiousity.  Why:
```
user = @robot.brain.userForId newmsg.u._id, name: newmsg.u.username
user.room = newmsg.rid
```

Instead of just setting the property like was being done below?

```
user = @robot.brain.userForId newmsg.u._id, name: newmsg.u.username, room: newmsg.rid
```